src/mixins/enum-mixin.ts

```diff
@@ -55,46 +65,74 @@ export default class EnumMixin extends Vue {
-  }
-
-  /** Returns True if item status is Correction Pending. */
-  isStatusCorrectionPending (item: any): boolean {
-    return item.status === FilingStatus.PENDING_CORRECTION
+    return (
+      (item.status === 'PENDING_ALTERATION') || // TODO: remove this workaround when API stops sending this
+      (item.status === 'PENDING_CORRECTION') || // TODO: remove this workaround when API stops sending this
+      (item.status === FilingStatus.PENDING)
+    )
   }
 
-  /** Returns True if item status is Alteration Pending. */
-  isStatusAlterationPending (item: any): boolean {
-    return item.status === FilingStatus.PENDING_ALTERATION
+  /** Returns True if item status is Withdrawn. */
+  isStatusWithdrawn (item: any): boolean {
+    return item.status === FilingStatus.WITHDRAWN
   }
 
   //
   // Filing Type helpers
   //
 
-  /** Returns True if task type is Alteration. */
+  /** Returns True if filing is an Alteration. */
   isTypeAlteration (item: any): boolean {
-    return (item.filingType === FilingTypes.ALTERATION)
+    return (item.name === FilingTypes.ALTERATION)
   }
 
-  /** Returns True if task type is Annual Report. */
+  /** Returns True if filing is an Annual Report. */
   isTypeAnnualReport (item: any): boolean {
-    return (item.filingType === FilingTypes.ANNUAL_REPORT)
+    return (item.name === FilingTypes.ANNUAL_REPORT)
   }
 
-  /** Returns True if task type is Correction. */
+  /** Returns True if filing is a Change of Address. */
+  isTypeChangeOfAddress (item: any): boolean {
+    return (item.name === FilingTypes.CHANGE_OF_ADDRESS)
+  }
+
+  /** Returns True if filing is a Change of Directors. */
+  isTypeChangeOfDirectors (item: any): boolean {
+    return (item.name === FilingTypes.CHANGE_OF_DIRECTORS)
+  }
+
+  /** Returns True if filing is a Change of Name. */
+  isTypeChangeOfName (item: any): boolean {
+    return (item.name === FilingTypes.CHANGE_OF_NAME)
+  }
+
+  /** Returns True if filing is a Correction. */
   isTypeCorrection (item: any): boolean {
-    return (item.filingType === FilingTypes.CORRECTION)
+    return (item.name === FilingTypes.CORRECTION)
   }
 
-  /** Returns True if task type is Name Incorporation Application. */
+  /** Returns True if filing is an Incorporation Application. */
   isTypeIncorporationApplication (item: any): boolean {
-    return (item.filingType === FilingTypes.INCORPORATION_APPLICATION)
+    return (item.name === FilingTypes.INCORPORATION_APPLICATION)
   }
 
-  /** Returns True if task type is Name Request. */
+  /** Returns True if filing is a Name Request. */
   isTypeNameRequest (item: any): boolean {
-    return (item.filingType === FilingTypes.NAME_REQUEST)
+    return (item.name === FilingTypes.NAME_REQUEST)
+  }
+
+  /** Returns True if filing is a Transition. */
+  isTypeTransition (item: any): boolean {
+    return (item.name === FilingTypes.TRANSITION)
+  }
+
+  /** Returns True is filing is a Staff Only filing. */
```

Comment:
Probably my typo here, but a type none the less 😁  `Returns true if..`

Reply:
Fixed.